src/EnergyPlus/SetPointManager.cc

```diff
@@ -5341,7 +5358,24 @@ namespace SetPointManager {
 			this->MySetPointCheckFlag = false;
 		}
 
-		this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
+		if ( CoolCoilInNode == 0 && CoolCoilOutNode == 0 ) {
+			this->SetPt = Node( RefNode ).TempSetPoint - ( Node( FanOutNode ).Temp - Node( FanInNode ).Temp );
```

Comment:
@lgu1234 When FreezeCheckEnable is false, I would expect to get the standard mixed air setpoint like this line.  But the block below is always forcing MinTemp is the coil node names are present.